### PR TITLE
Configure the Secure API for embed Signer

### PR DESCRIPTION
**Pull request by ngotchac**
Fixes https://github.com/ethcore/parity/issues/4885

### CODE REVIEW

File: web3/index.js [hunk 1]
```diff
@@ -89,3 +95,17 @@ function loadScripts () {
...
 }
 
+function configureApi (config) {
+  const { DAPPS } = config;
+
+  const dappsInterface = DAPPS.split(':')[0];
+  const dappsPort = DAPPS.split(':')[1];
+
+  if (dappsInterface) {
+    window.secureApi._dappsInterface = dappsInterface;
```
Comment (tomusdrw) [inline web3/index.js]: A bit hacky to modify private fields. Can we expose this as a method in new versions of Parity?

**Review by tomusdrw — CHANGES_REQUESTED** [review]

File: web3/index.js (hunk 1 above)
Comment (ngotchac) [reply web3/index.js]: Yes, it is hacky :/ Would be better to expose those from the API, but this means that users must update to a new version of the UI. Will implement it though

Comment (tomusdrw) [reply web3/index.js]: Yeah. Let's use proper public method for new versions and that hack for old ones (i.e. if the method is not available)

Comment (ngotchac) [reply web3/index.js]: Updated (needs paritytech/parity#4922)

**Review by tomusdrw — APPROVED** [review]

**Pull request merged**